practice-one/src/styles/sections/layout.css

```diff
@@ -1,9 +1,14 @@
 .container {
   margin-inline: auto;
   padding: 70px;
-  width: 1440px;
 }
 
 html, body {
   font-family: var(--font-family-primary);
 }
+
+@media (max-width: 429px) {
+  .container {
+    padding: 24px;    
+  }
+}
```

Comment:
What about screen tablet?

Reply:
In original screen don't have the design for tablet screen

Reply:
I see, but we need to make sure everything is broken right? Simply put a screenshot for tablet
This is your target:
```md
Create a responsive website :
Desktop breakpoint (1440px): Match with design
Mobile breakpoint(428px): Match with design
Another breakpoints: Avoid to break application
```

Reply:
I changed it ,thanks